web/src/styles/globals.css

```diff
@@ -2,6 +2,11 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
+html, body {
+    @apply overflow-x-hidden scroll-smooth;
```

Comment:
Not a fan of `overflow-x-hidden` here; if there's overflow, I'd like to see the scrollbars so we can fix it.

Reply:
This is kind of a hack when displacing (transitioning out of the screen) an element. It hides it on the sides and prevents a scrollbar from being added. I don't like using overflow neither as it can introduce issues with other css rules like position: sticky but in this case it was necessary. 

What I'm going to do is just quickly see if it is possible to do the x-transition without overflow else I'll just do a scale transition for that element that is overflowing

Reply:
Makes sense.